src/components/layout.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { Link, Outlet } from 'react-router-dom';
+import './layout.css';
+
+export default function Navbar() {
+  const [data, setData] = useState({
+    render: true,
+    class: 'links',
+  });
+
+  function toggle() {
+    const { render } = data;
+    if (render === false) {
+      setData(() => ({
+        render: true,
+        class: 'links',
+      }));
+    } else {
+      setData(() => ({
+        render: false,
+        class: 'links hide',
+      }));
+    }
+  }
+
+  return (
+    <>
+      <header className="navbar">
+        <div className="navdiv">
+          <h1 className="header">Math Magicians</h1>
```

Comment:
- [ ] Is this Math Magicians or Bookstore project? Please 🙏🏽 use the correct title for this milestone.